.github/workflows/go-test-config/action.yaml

```
@@ -0,0 +1,174 @@
+name: go-test
+description: "test kubectl-rook-ceph commands"
+inputs:
+  op-ns:
+    description: operator namespace where rook operator will deploy
+    required: false
+  cluster-ns:
+    description: cluster namespace where ceph cluster will deploy
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: set environment variables
+      run: |
+        OP_NS_OPT=""
+        CLUSTER_NS_OPT=""
+        test -n "${{ input.op-ns }}" && OP_NS_OPT="--operator-namespace ${{ input.op-ns }}"
```

```suggestion
        test -n "${{ input.op-ns }}" && OP_NS_OPT="--operator-namespace ${{ input.op-ns }}"
```

`test -n` here is for?

---

It's old code. OP_NS_OPT should be set iff input.op-ns isn't rook-ceph. CLUSTER_NS_OPT should also be set iff input.cluster-ns isn't rook-ceph. I'll fix them.